### ISSUE TITLE
Translation into Russian

### ISSUE THREAD

**Issue by Nicolayka** [zmilla93/SlimTrade#85]
Hi!
I can help you translate the program into Russian language

**Comment by zmilla93** [comment]
That would be awesome!

To clarify, SlimTrade should work with the Russian client and with Russian trades. Is that correct?

Unfortunately the program currently doesn't support other languages. Doing so requires me to define every piece of text in a separate language file, then redefine that language file for every language I want to support. Setting up and maintaining this is a bit of a pain. I'm willing to, just not right now. I'll revisit this idea closer to a v1.0.0 release when things are more stable, which is steadily approaching.

**If other people have have interest in support for other languages in general (especially if you are willing to translate), consider reply/reacting to this issue so I know there is demand!**

**Comment by Nicolayka** [comment]
> To clarify, SlimTrade should work with the Russian client and with Russian trades. Is that correct?
Yes, that's right. But we just need to take into account that Russian players can trade with both Russian-speaking and English-speaking players :)

As soon as a translation is required, I am ready to help! Ping me here and I'll come running to help!
